test/jit/test_backends.py

```diff
@@ -685,3 +687,75 @@ def test_execution(self):
     @skipIfRocm
     def test_errors(self):
         self.error_module_compiler_test.test_errors()
+
```

Comment:
Should we move these tests to a separate file?

Thus far the tests in text_backends.py are not backend specific, and focus on the mechanism rather than any specific backend implementation.
Thus, I would prefer having either a single test file per backend (test_nnapi_backend.py), or if you think many backends can share a common test class then have one file (unfortunately the better name test_backends.py is used, so maybe test_backend_impls.py).

What do you think @iseeyuan ?

Reply:
Makes sense to me. Probably it could also resolve the issue of int 32 type issue.

Reply:
It did not resolve the issue, sadly.